Tools/CISupport/build-webkit-org/factories.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, platform, configuration, architectures, triggers=None, additi
 
             self.addStep(ArchiveBuiltProduct())
             self.addStep(UploadBuiltProduct())
-            if platform.startswith('mac') or platform.startswith('ios-simulator') or platform.startswith('tvos-simulator') or platform.startswith('watchos-simulator'):
+            if platform.startswith('mac') or platform.startswith('ios') or (platform.startswith('watchos') and not platform.startswith('watchos-simulator')):
```

Comment:
This probably wouldn't work as expected since this code is inside `if triggers:` condition.

Also, just to clarify, do our users need these archives uploaded for every revision we build, or are they ok with only manually triggering with specified hash and getting the build product for those builds only?

Finally, do we need both the full archive and minified archive?

Reply:
> Getting a specified hash / identifier would be sufficient.

ok, I can take care of that.

Reply:
Ok, thanks, I'll close this PR.